src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -818,6 +818,7 @@ private void handleOne(final Callback[] callbacks, final int idx) throws IOExcep
                 } else if (callback instanceof CredentialCallback) {
                     final CredentialCallback credentialCallback = (CredentialCallback) callback;
 
+                    // TODO: should consider realm from RealmCallback
```

Comment:
I assume you're referring to org.wildfly.security.password.spec.DigestPasswordAlgorithmSpec here?